<?xml version="1.0"?>
<testsuites>
 <testsuite name="index2_script" package="recursion.suite" timestamp="2019-03-11 09:56:36.864 -0600"
 tests="1"  failures="1" errors="0" time="0.621300">
  <properties>
   <property name="RunDate" value="2019-03-11 09:56:36.864 -0600"/>
   <property name="Status" value="Failure"/>
  </properties>
  <testcase name="index2.script" assertions="1" status="failure"  time="0.619569">
    <failure type="FAILURE" message="STMaxCallDepthExceeded Handlers were called to a depth exceeding the maximum limit (40). HINT: One common cause for this error is a script that has been given the same name as a built-in command or function unintentionally calling itself. -- Execution Time 0:00:00 index2.script" />
  </testcase>
 </testsuite>
</testsuites>
